Misc/NEWS.d/next/Security/2019-02-08-13-58-03.bpo-35926.uIEgyE.rst

```diff
@@ -0,0 +1 @@
+Update to OpenSSL 1.1.1a for Windows.

```

Comment:
Looks like this news file has been superseded by the one for 1.1.1b and should be deleted.

Reply:
Thanks! This should be fixed now